test/e2e/main/main_test.go

```diff
@@ -1450,3 +1450,61 @@ func createCacheWithCR(cacheName string, nameSpace string, clusterName string) *
 		},
 	}
 }
+
+// Test if the cluster made up of 2 nodes is working correctly after it was restarted by OutOfMemory exception
+func TestPodDegradationAfterOOM(t *testing.T) {
+	t.Parallel()
+
+	//Creating Infinispan cluster
+	ispn := tutils.DefaultSpec(testKube)
+	ispn.Name = strcase.ToKebab(t.Name())
+	ispn.Labels = map[string]string{"test-name": t.Name()}
+	ispn.Spec.Replicas = 2
+	ispn.Spec.Container.Memory = "512Mi"
```

Comment:
This is default value, I would try going a bit lower to speed up test. Eg.: 384Mi or maybe even 256Mi depending on how empty node needs

Reply:
Lowered memory to 256, it goes in OOM faster